vuepress/docs/next/docs/compose/local-hub-overview.md

```diff
@@ -1,67 +1,53 @@
 ---
 sidebarDepth: 2
 ---
-# Local Hub
+# Entando Local Hub
 
 ## Introduction
 
-The Local Hub is a repository of modular components that can be reused within an Entando Application.
+The Entando Local Hub is a repository of modular components accessible from an Entando App Builder. It represents the Entando Bundles deployed or installed in the Entando Application.
```

Comment:
Like before, represents just seems the wrong word here. None of these definitions of represent apply https://www.merriam-webster.com/dictionary/represent